sdks/python/apache_beam/pvalue.py

```diff
@@ -158,6 +158,10 @@ def __ne__(self, other):
   def __hash__(self):
     return hash((self.tag, self.producer))
 
+  def __class_getitem__(cls, type_param):
```

Comment:
I believe this method is unnecessary, since Generic[T] implements something similar. You can can inspect a PCollection to get its arguments.
For example:
```
>>> import typing
>>> T = typing.TypeVar('T')
>>> class A(typing.Generic[T]):
...   pass
... 
>>> A[int].__args__
(<class 'int'>,)
```

Reply:
Thanks, fixed.